argocd_website/static/src/js/portal.js

```diff
@@ -42,4 +42,74 @@ odoo.define("argocd_website.portal", function (require) {
         },
     });
 
+    publicWidget.registry.DomainCNAMECheck = publicWidget.Widget.extend({
+        selector: ".o_portal_wrap .js_domain_cname_check",
+        events: {
+            "click button": "_onButtonClick",
+            "keydown input": "_onKeyDown",
+        },
+        start: function () {
+            this._super();
+            this.$input = this.$el.find("input");
+            this.$validFeedback = this.$el.find(".text-success");
+            this.$invalidFeedback = this.$el.find(".text-danger");
+            this.$button = this.$el.find("button");
+            this.$buttonIcon = this.$button.find("i");
+            this.appId = parseInt(this.$el.attr("data-app-id"), 10);
+            this.tagId = parseInt(this.$el.attr("data-tag-id"), 10);
+            this.subdomain = this.$el.attr("data-subdomain");
+
+            this.checkDomainCNAME();
+        },
+
+        _onButtonClick: function () {
+            this.checkDomainCNAME();
+        },
+
+        _onKeyDown: function () {
+            this.$input.removeClass("is-invalid");
+            this.$input.removeClass("is-valid");
+        },
+
+        checkDomainCNAME: function () {
+            var self = this;
+            var domain = this.$input.val();
+
+            if (!domain) {
+                self.$input.removeClass("is-invalid");
+                self.$input.removeClass("is-valid");
```

Comment:
you can include both the classes at once separated by space removeClass("is-invalid is-valid")